src/smolagents/monitoring.py

```diff
@@ -124,7 +124,7 @@ def log_markdown(self, content: str, title: Optional[str] = None, level=LogLevel
         else:
             self.log(markdown_content, level=level)
 
-    def log_code(self, title: str, content: str, level: int = LogLevel.INFO) -> None:
+    def log_code(self, title: str, content: str, level: str | LogLevel = LogLevel.INFO) -> None:
```

Comment:
I thought this would be `int | LogLevel`.

Reply:
hmm self.log accept string and convert it in the LogLevel.